docs/root/start/start.rst

```diff
@@ -160,6 +160,23 @@ for controlling access to an ``envoy`` socket from outside of the container.
 
 If you wish to run the container as the ``root`` user you can set ``ENVOY_UID`` to ``0``.
 
+The ``envoy`` image sends application logs to ``/dev/stdout`` and ``/dev/stderr`` by default, and these
+can be viewed in the container log.
+
+If you send application, admin or access logs to a file output, the ``envoy`` user will require the
+necessary permissions to write to this file. This can be achieved by setting the ``ENVOY_UID`` and/or
+by making the file writeable by the envoy user.
+
+For example, to mount a log folder from the host and make it writable, you can:
+
+  $ mkdir logs
```

Comment:
These lines are rendering poorly. I think you need to add `.. substitution-code-block:: none` above the 3 lines. There's an example of that earlier in the file.

Reply:
added